packages/nc-gui/components/cell/attachment/index.vue

```diff
@@ -42,6 +43,8 @@ const currentCellRef = inject(CurrentCellInj, dropZoneInjection.value)
 
 const isLockedMode = inject(IsLockedInj, ref(false))
 
+const isDrawerExist = inject(IsExpandedFormOpenInj, ref())
```

Comment:
@sreehari2003 Nitpick: Try to keep the injection variable name close to the injection state name